internal/tsc_wrapped/compiler_host.ts

```diff
@@ -130,8 +130,8 @@ export class CompilerHost implements ts.CompilerHost, tsickle.TsickleHost {
 
   /** Allows suppressing warnings for specific known libraries */
   shouldIgnoreWarningsForPath(filePath: string): boolean {
-    return this.bazelOpts.ignoreWarningPaths.some(
-        p => !!filePath.match(new RegExp(p)));
+    return (this.bazelOpts.ignoreWarningPaths || [])
```

Comment:
is this change required for it to work?

Reply:
No, but it is a bug in rules_typescript that we might as well merge in. I ran into it frequently while doing ABC work

Reply:
hmm, I'd rather keep it separate, and also understand why this is sometimes not set (I think it's easier to understand if the data is not nullable)